japicmp/src/main/java/japicmp/compat/CompatibilityChanges.java

```diff
@@ -66,7 +66,9 @@ private void evaluateBinaryCompatibility(JApiClass jApiClass, Map<String, JApiCl
 		if (hasModifierLevelDecreased(jApiClass)) {
 			addCompatibilityChange(jApiClass, JApiCompatibilityChange.CLASS_LESS_ACCESSIBLE);
 		}
-		checkIfGenericTemplatesHaveChanged(jApiClass);
+		if (jApiClass.getChangeStatus() != JApiChangeStatus.NEW) {
```

Comment:
```suggestion
		if (jApiClass.getChangeStatus() != JApiChangeStatus.MODIFIED) {
```
Maybe this'd makes more sense.